yarn.lock

```diff
@@ -4083,6 +4078,11 @@ map-obj@^2.0.0:
   resolved "https://registry.yarnpkg.com/map-obj/-/map-obj-2.0.0.tgz#a65cd29087a92598b8791257a523e021222ac1f9"
   integrity sha1-plzSkIepJZi4eRJXpSPgISIqwfk=
 
+marked@^1.0.0:
```

Comment:
Yea, this might not work.

We should briefly investigate what plugins are out there and figure out how easy they are to integrate and how they mix with the rest of our architecture.

Here's a couple I found, there's probably more:
https://github.com/metonym/svelte-highlight
https://www.npmjs.com/package/svelte-prism

Reply:
Why would `marked` not work?

Reply:
The [@html](https://svelte.dev/docs#html) directive should allow for rendering of the computed markdown.

Reply:
It could work, but would require a bit more integration work than just adding the package. But yea, the PR is still in draft, maybe I should have waited with my remark.

Reply:
It might be as trivial as:

``` svelte
{@html marked(issue.description)}
```

Reply:
Check 9f32bfc

Reply:
Dope!